pkg/plugin/linuxutil/ethtool_stats_linux.go

```diff
@@ -29,10 +33,12 @@ func NewEthtoolReader(opts *EthtoolOpts, ethHandle EthtoolInterface) *EthtoolRea
 		}
 	}
 	return &EthtoolReader{
-		l:         log.Logger().Named(string("EthtoolReader")),
-		opts:      opts,
-		data:      &EthtoolStats{},
-		ethHandle: ethHandle,
+		l:           log.Logger().Named(string("EthtoolReader")),
+		opts:        opts,
+		data:        &EthtoolStats{},
+		ethHandle:   ethHandle,
+		unsupported: make(map[string]bool),
+		limit:       limitValue,
```

Comment:
Can't this be pulled in from `opts`? Why not make it tunable and set defaults at the config level?